go-controller/pkg/cluster/cluster.go

```diff
@@ -34,7 +34,14 @@ const (
 	// OvnHostSubnet is the constant string representing the annotation key
 	OvnHostSubnet = "ovn_host_subnet"
 	// OvnClusterRouter is the name of the distributed router
-	OvnClusterRouter = "ovn_cluster_router"
+	OvnClusterRouter              = "ovn_cluster_router"
+	OvnNodeGatewayMode            = "ovn_node_gateway_mode"
+	OvnNodeVlanID                 = "ovn_node_vlan_id"
+	OvnNodeIfaceID                = "ovn_node_iface_id"
+	OvnNodeMacAddress             = "ovn_node_mac_address"
+	OvnNodeSubnet                 = "ovn_node_subnet"
```

Comment:
`OvnNodeSubnet` is actually already known by the master at the point when it should sync the gateway logical network (eg in pkg/cluster/master.go::addNode()) so it doesn't need to be an annotation.